src/server/debug.rs

```diff
@@ -266,6 +266,43 @@ impl Debugger {
         }
     }
 
+    pub fn do_unsafe_recover(&self, region_id: u64) -> Result<()> {
+        let db = &self.engines.kv_engine;
+        let key = keys::region_state_key(region_id);
+        let mut region_state = box_try!(db.get_msg_cf::<RegionLocalState>(CF_RAFT, &key)).unwrap();
+        if region_state.state == PeerState::Tombstone {
+            return Ok(());
+        }
+
+        let store_id = self.get_store_id()?;
+        let peers = region_state.mut_region().take_peers();
+        let peers_len = peers.len() as u64;
+        let peer = match peers.into_iter().find(|p| p.get_store_id() == store_id) {
+            Some(p) => p.clone(),
+            None => {
+                let err_msg = "RegionLocalState doesn't contains the peer itself";
+                return Err(Error::Other(err_msg.into()));
+            }
+        };
+
+        let conf_ver = region_state.get_region().get_region_epoch().get_conf_ver();
+        let mut new_epoch = RegionEpoch::new();
+        new_epoch.set_conf_ver(conf_ver + peers_len - 1);
```

Comment:
The region on the peer maybe is behind latest info on `pd`. So we should add a value to `conf_ver` to make it larger than pd. So I add `peers_len - 1`, because if we remove peers one by one, we need remove `peers_len - 1` times.

Reply:
Oh, I see. Please add some comments about the formula.